### ISSUE TITLE
SUB - Colour contrast calculation does not take polarity into account

### ISSUE THREAD

**Issue by Phil0menaA** [w3c/matf#116]
Hello everyone,

I am new here, but understand this platform is open to contributions from the public, so here goes:

The current colour contrast calculation method does not fully reflect human visual perception as it does not take into account:

1) The contrast polarity: Currently contrast is calculated symmetrically, but light text on dark backgrounds appears thinner due to glow, affecting readability. The [APCA model](https://apcacontrast.com/) for calculation seems to offer an interesting solution to this problem.
2) Visual vibration: Certain color combinations cause visual fatigue and increased mental workload (as per [this study](https://www.sciencedirect.com/science/article/abs/pii/S0141938224000313)), which is not accounted for.

Some examples of the issue:
![Image](https://github.com/user-attachments/assets/ef5251eb-24f7-4bcf-ad93-5ff27a36e41f)
![Image](https://github.com/user-attachments/assets/f37f327f-a226-4b23-9859-ba05527012df)

Others have written about this too:
https://khan-tw.medium.com/wcag2-are-you-still-using-it-ui-contrast-visibility-standard-readability-contrast-f34eb73e89ee
https://typefully.com/DanHollick/wcag-3-and-apca-sle13GMW2Brp
https://uxdesign.cc/dont-rely-only-on-tools-to-pick-accessible-colour-combinations-48c12ff4a192

I think WCAG 3 should be the opportunity to fix this. :)
